cob_image_flip/ros/src/image_flip.cpp

```diff
@@ -460,8 +527,8 @@ void ImageFlip::pcCallback(const sensor_msgs::PointCloud2::ConstPtr& point_cloud
 		{
 			for (int u = 0; u < point_cloud_out_msg->width; u++)
 			{
-				int src_u = rot_mat_inv.at<double>(0,0)*u + rot_mat_inv.at<double>(0,1)*v + rot_mat_inv.at<double>(0,2);
```

Comment:
Just leave it the old way in this case, because the rotation matrix is always the same format CV_64FC1.